jwt.js

```diff
@@ -432,17 +452,25 @@ function fastifyJwt (fastify, options, next) {
   }
 
   function requestVerify (options, next) {
-    let useLocalVerifier = true
+    const request = this
 
-    if (typeof options === 'function' && !next) {
+    if (next === undefined) {
```

Comment:
Just moving this from below to preserve options passed in even when using promises.